<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res.qrc">
    <normaloff>:/ico/res/kawaii-emoji-messenger.png</normaloff>:/ico/res/kawaii-emoji-messenger.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>395</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Navigation">
    <property name="title">
     <string>&amp;Navigation</string>
    </property>
    <addaction name="actionNavReload"/>
    <addaction name="separator"/>
    <addaction name="actionNavCopyCurrentUrl"/>
    <addaction name="separator"/>
    <addaction name="actionNavBack"/>
    <addaction name="actionNavNext"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelpDonate"/>
    <addaction name="actionBugReport"/>
    <addaction name="separator"/>
    <addaction name="actionHelpAbout"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionToolsSettings"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Navigation"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionHome"/>
   <addaction name="actionMessages"/>
   <addaction name="actionNotifications"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>200</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTextEdit" name="textEditChat">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="documentTitle">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushEmojiView">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/icons/res/smile.png</normaloff>:/icons/res/smile.png</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushSend">
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionHome">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/res/home.png</normaloff>:/icons/res/home.png</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="actionMessages">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/res/message.png</normaloff>:/icons/res/message.png</iconset>
   </property>
   <property name="text">
    <string>Messages</string>
   </property>
  </action>
  <action name="actionNotifications">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/res/notifications.png</normaloff>:/icons/res/notifications.png</iconset>
   </property>
   <property name="text">
    <string>Notifications</string>
   </property>
   <property name="toolTip">
    <string>Notifications</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionNavBack">
   <property name="text">
    <string>Back</string>
   </property>
   <property name="toolTip">
    <string>Go to previous page</string>
   </property>
  </action>
  <action name="actionNavNext">
   <property name="text">
    <string>Next</string>
   </property>
   <property name="toolTip">
    <string>Return to next page</string>
   </property>
  </action>
  <action name="actionNavReload">
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="toolTip">
    <string>Reload page</string>
   </property>
  </action>
  <action name="actionNavCopyCurrentUrl">
   <property name="text">
    <string>Copy current url</string>
   </property>
   <property name="toolTip">
    <string>Copy current url on the clipboard</string>
   </property>
  </action>
  <action name="actionToolsSettings">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Open the settings dialog</string>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>Show about dialog</string>
   </property>
  </action>
  <action name="actionHelpDonate">
   <property name="text">
    <string>Donate</string>
   </property>
   <property name="toolTip">
    <string>Support the project donating some money</string>
   </property>
  </action>
  <action name="actionShowHideWindow">
   <property name="text">
    <string>Hide</string>
   </property>
  </action>
  <action name="actionBugReport">
   <property name="text">
    <string>Report a bug...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
